examples/postgres/flyway_postgres_test.go

```diff
@@ -0,0 +1,164 @@
+package main
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"github.com/testcontainers/testcontainers-go"
```

Comment:
I think this line should be moved next to the testcontainers imports

Reply:
there is something odd about the way imports are organised ... I've seen projects which are alphabetical and ones which are more "golang" default. It must be a project setting as I don't manually do anything.

Reply:
see if I organise imports I get get

<img width="636" alt="image" src="https://github.com/CyberOwlTeam/testcontainers-go-flyway/assets/6204676/18fe2a02-00db-46f9-ab5a-4ebc833b440f">

Reply:
I'm a bit confused by how this is setup tbh ... I played around with it by manually moving stuff, deleted the spaces, and then it miraculously started organising them in the way I would expect